aria/storage/modeling/instance_elements.py

```diff
@@ -193,7 +193,7 @@ def service_instance(cls):
         return cls.many_to_one_relationship('service_instance')
 
     @declared_attr
-    def interfaces(cls):
+    def interface(cls):
```

Comment:
One to many. An interface has zero or more operations, but each operation has only one parent interface.